cmd/fleet/cron.go

```diff
@@ -714,11 +715,14 @@ func startCleanupsAndAggregationSchedule(
 				return ds.UpdateOSVersions(ctx)
 			},
 		),
-	).Start()
+	)
+	s.Start()
+
+	return s, nil
 }
 
-func startSendStatsSchedule(ctx context.Context, instanceID string, ds fleet.Datastore, config config.FleetConfig, logger kitlog.Logger) {
-	schedule.New(
+func startSendStatsSchedule(ctx context.Context, instanceID string, ds fleet.Datastore, config config.FleetConfig, license *fleet.LicenseInfo, logger kitlog.Logger) (*schedule.Schedule, error) {
```

Comment:
We discussed having these `start*` would be turned into something like the same name without the `start` probably and just return s (without starting it) and have the cron schedules handle the starting, since it also handles the stopping.

Did that idea not pan out when it came time to do it?

Reply:
IIRC we discussed adding the CronScheduleStarterFunc [type](https://www.loom.com/share/35832735f2d74b45991fe590efaef439) but I don't recall moving the `s.Start()` calls out of the schedule package. If you think it makes more sense, I can refactor. It'll impact how some of the tests are structured, but probably not a huge deal either way.

Reply:
I think having the Cron Schedules be the thing responsible for starting and stopping schedules is cleaner arch than having schedules start over here and stop over there. But if this delays this PR too much, I would say let's just do it later.

Reply:
Ok, I'll take a look. 

Note that stopping the schedules is really being handled via the cancelFunc for the server context, and isn't being controlled inside the `cronSchedulesService` beyond a simple pass through of the server context into the starter function.

The `shutdownCronSchedulesService` is just cleaning up the stats for any pending jobs so I'll rename it `cleanupCronStatsOnShutdown` to clarify that.